docs/extend/extend-sdks.md

```diff
@@ -0,0 +1,6 @@
+#
+The Zowe SDKs are open source. You can contribute to add features, enhancements, and bug fixes to the source code.
+
+For example, the functionality is currently limited to interfaces provided by IBM z/OSMF. You could enhance the SDK to work with other services, such as IBM CICS.
```

Comment:
> You could enhance the SDK to work with other services, such as IBM CICS.

The CICS SDK technically already exists in the CICS plugin repo, but has not yet been published separately (same situation for other CLI plugins). Perhaps instead we could say something like:
> If you are a CLI plugin developer, you could enhance the SDK by creating an SDK package to expose the programmatic APIs for your service.